pkg/chunkenc/symbols.go

```diff
@@ -400,7 +435,10 @@ func symbolizerFromEnc(b []byte, pool compression.ReaderPool) (*symbolizer, erro
 				return nil, err
 			}
 		}
-		s.labels = append(s.labels, string(buf))
+		label := string(buf)
+		idx := uint32(i)
+		s.labels = append(s.labels, label)
+		s.symbolsMap[label] = idx
```

Comment:
tiny nit: extracting the conversions into variables that are only used once is a little distracting here 🤷 

(ditto with the change on line 352)